src/components/step/StepDetailText/StepDetailText.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react'
+import stepElementStyle from '../step-helpers';
+import './StepDetailText.css'
+import { StepElementProps } from '../shared';
+import StepMarkdown from '../StepMarkdown/StepMarkdown';
+
+
+export default function (props: StepElementProps) {
+    if (!props.text) {
+      return (<></>)
+    }
+    return (
+      <div id="detail-text" className="step-detail-text" style={stepElementStyle(props)}>
```

Comment:
Does this need an id?  Good chance it could be used multiple times on a page so would not be unique